toolchain/semantics/semantics_ir.cpp

```diff
@@ -5,13 +5,19 @@
 #include "toolchain/semantics/semantics_ir.h"
 
 #include "common/check.h"
-#include "llvm/Support/FormatVariadic.h"
-#include "toolchain/lexer/tokenized_buffer.h"
+#include "toolchain/semantics/node_kind.h"
+#include "toolchain/semantics/node_ref.h"
+#include "toolchain/semantics/node_store.h"
+#include "toolchain/semantics/nodes/binary_operator.h"
+#include "toolchain/semantics/nodes/function.h"
+#include "toolchain/semantics/nodes/integer_literal.h"
+#include "toolchain/semantics/nodes/return.h"
+#include "toolchain/semantics/nodes/set_name.h"
```

Comment:
I think node_store.h _must_ include all individual nodes; I wasn't duplicating #includes as a consequence.

Reply:
There is only a practice here. We can technically not putting any headers and get away with it by Bazel. Given what are included from other files, we may want to see where each Print comes. Also, I am aware of classic programmers who may use good old tools and I want them to freely index it.